build_docker.sh

```diff
@@ -10,6 +10,7 @@ cd ../
 
 docker build -t server . -f ./app/Dockerfile --build-arg GIT_HASH=$GIT_HASH
 docker build -t proxy ./proxy
+docker build -t eth_processor . -f ./eth_worker/Dockerfile --build-arg CONTAINER_TYPE=PROCESSOR
 docker build -t low_priority_eth_worker . -f ./eth_worker/Dockerfile --build-arg CONTAINER_TYPE=LOW_PRIORITY_WORKER
 docker build -t high_priority_eth_worker . -f ./eth_worker/Dockerfile --build-arg CONTAINER_TYPE=HIGH_PRIORITY_WORKER
```

Comment:
why do these all need seperate builds? Can't we just inject arguments at runtime?